packages/qdrant-loader-core/src/qdrant_loader_core/graph/schema/init_schema.py

```
@@ -0,0 +1,109 @@
+        await _set_version(graph_store, version)
+
+        logger.info(
+            "Graph schema migration v%s applied successfully",
+            version,
+        )
+
+
+# ------------------------
+# INDEXES
+# ------------------------
+async def _ensure_indexes(graph_store):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_ensure_indexes`

Add return type annotation: `None`

(ANN202)